homeassistant/components/ecobee/sensor.py

```diff
@@ -19,22 +19,49 @@
 
 SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
     SensorEntityDescription(
-        key="temperature",
-        name="Temperature",
-        native_unit_of_measurement=TEMP_FAHRENHEIT,
-        device_class=SensorDeviceClass.TEMPERATURE,
-        state_class=SensorStateClass.MEASUREMENT,
+        key = "temperature",
+        name = "Temperature",
+        native_unit_of_measurement = TEMP_FAHRENHEIT,
+        device_class = SensorDeviceClass.TEMPERATURE,
+        state_class = SensorStateClass.MEASUREMENT,
     ),
     SensorEntityDescription(
-        key="humidity",
-        name="Humidity",
-        native_unit_of_measurement=PERCENTAGE,
-        device_class=SensorDeviceClass.HUMIDITY,
-        state_class=SensorStateClass.MEASUREMENT,
+        key = "humidity",
```

Comment:
Looks like a lot of these got reformatted, probably want to format them as before

Reply:
Fixed